tokio/tests/io_read.rs

```diff
@@ -1,41 +1,38 @@
 #![deny(warnings, rust_2018_idioms)]
+#![feature(async_await)]
```

Comment:
we could prob just throw this in a mod under the read.rs?

Reply:
I think it has to be in each test file.

Reply:
sorry I mean, the whole test.